problems/arrayMethods.js

```diff
@@ -145,7 +228,14 @@ const numberTimesIdx = () => {};
  * @returns {boolean} Are all numbers positive, even, and not ending in 0.
  */
 
-const arePositiveEvenAndNonZeroEnding = () => {};
+const arePositiveEvenAndNonZeroEnding = (nums) => {
+  return nums.every((el) => {
+    if (el >= 0 && el % 2 === 0 && el % 10 !== 0) {
```

Comment:
You can also return this code on line 233, as opposed to making this an if statement and returning true below.

Reply:
as in "return if (el...." ?

Reply:
return (el...)